.github/dependabot.yml

```diff
@@ -6,35 +6,54 @@ updates:
       interval: "daily"
     allow:
       - dependency-type: "all"
+    groups:
+      production-dependencies:
+        dependency-type: "production"
+      development-dependencies:
+        dependency-type: "development"
+    open-pull-requests-limit: 3
+  - package-ecosystem: "gomod"
+    directory: "/"
+    target-branch: "release-v0.19"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-type: "direct"
+    open-pull-requests-limit: 0
```

Comment:
If we set this to 0, do we need the configuration for this and others branches at all?

Reply:
yes, because this should not affect security issue PRs.
```
This option has no impact on security updates, which have a separate, internal limit of ten open pull requests.
```
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

Reply:
From the documentation, I've understood that the security update PRs are only opened for the default branch:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#configuration-options-for-the-dependabotyml-file

> Security updates are raised for vulnerable package manifests only on the default branch. When configuration options are set for the same branch (true unless you use target-branch), and specify a package-ecosystem and directory for the vulnerable manifest, then pull requests for security updates use relevant options.